### DISCUSSION TITLE
Adding subtabs

### DISCUSSION THREAD

**Discussion by lkoumis** [tabpanelwidget/tabpanelwidget#8] [Ideas] [answered]
Great work. Wondering if under each tab,  add subtabs (horizontally) similar to this:
https://svctenvims.dot.ca.gov/DEA_Library/

**Comment by thierryk** [comment, marked as answer]
I'd never tried that before today as I'm not sure this is a good idea regarding UX but it turns out it's something that is pretty easy to do:

https://codepen.io/thierry/full/BaZodQX

**Reply by lkoumis** [reply]
Fantastic. Thanks. After looking at the changes, it was simple.
